test/integration/helpers/getTrackers.test.js

```diff
@@ -0,0 +1,43 @@
+const getTrackers = require('../../../src/helpers/getTrackers')
+
+const address = '0xBFCF120a8fD17670536f1B27D9737B775b2FD4CF'
+const config = 'TrackerRegistryDev.json'
+const jsonRpcProvider = 'http://localhost:8545'
+
+describe('getTrackers', () => {
+    test('get array of trackers', async () => {
+        const trackers = await getTrackers(address, config, jsonRpcProvider)
+        expect(trackers).toStrictEqual([
+            'ws://10.200.10.1:30301',
+            'ws://10.200.10.1:30302',
+            'ws://10.200.10.1:30303'
+        ])
+    })
+
+    test('throw exception if address is wrong', async (done) => {
+        try {
+            await getTrackers('address', config, jsonRpcProvider)
```

Comment:
[Nitpick] In this test case you may want to use an Ethereum address, e.g. `0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF` because now it is trying to use ENS (Ethereum name service) to lookup address of name 'address'. Or maybe have two tests: one for ENS and one for concrete address.